client/src/components/CartHeader/CartHeader.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import CartHeader from "./CartHeader";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("CartHeader Component Tests", () => {
+  const props = {};
+
+  it("CartHeader renders correctly", () => {
```

Comment:
can add more tests now:
1. Cart header shows restaurant name
2. Cart header contains scan button
3. Cat header scan button, on click opens scanner

Reply:
tests added in #159, for (3) we test that clicking the button in header will call the callback. However, since this button callback is passed as a prop to CartHeader, we will defer testing whether callback launches Scan flow into ScanStore page tests instead.